src/sound_openal.cpp

```diff
@@ -265,6 +284,46 @@ struct PlayingSound
 	bool loop;
 };
 
+class SoundManagerGlobal
+{
+public:
+	unique_ptr_alcdevice  m_device;
+	unique_ptr_alccontext m_context;
+public:
+	SoundManagerGlobal() :
```

Comment:
if it's global it's a Singleton, not "Global". Please rename it

Reply:
We already have a SoundManager class (ISoundManager). The SoundManagerGlobal is not a singleton of (I)SoundManager, it is a separate class, so hopefully SoundManagerSingleton (after rename) is not confusing.